mailing_list/management/commands/sync_listserv.py

```diff
@@ -18,20 +19,18 @@ def __init__(self, *args, **kwargs):
     def handle(self, *args, **options):
         logger.info("Beginning sync_listserv job for canvas_course_ids %s", args)
         try:
-            mailing_lists = []
             canvas_course_ids = []
             if args:
                 for ml in MailingList.objects.filter(canvas_course_id__in=args):
                     canvas_course_ids.append(ml.canvas_course_id)
-                    mailing_lists.append(ml)
+                    ml.sync_listserv_membership()
             else:
                 for ml in MailingList.objects.all():
                     canvas_course_ids.append(ml.canvas_course_id)
-                    mailing_lists.append(ml)
+                    ml.sync_listserv_membership()
 
-            logger.info("Synchronizing listserv mailing lists for canvas course ids %s", canvas_course_ids)
-
-            sync_mailing_lists_with_listserv(mailing_lists)
```

Comment:
I just meant that previously, we found the set of lists to sync, then synced them in one call.  Now we're syncing them as we find them.  I was wondering if we cared whether we hit an error at list 2 of 4.  I don't think we do, just wanted to clarify.